packages/visual-dev/CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [1.3.0] - 2023-1-16
```

Comment:
Changelog version does not match what is currently in the package.json @noahwc just flagging to make sure everything is lining up before launch